docs/metrics.md

```diff
@@ -33,10 +33,10 @@ an hashed version of the Google Advertising ID.
 
 The following metrics are added to the ping:
 
-| Name | Type | Description | Data reviews | Extras | Expiration |
```

Comment:
delet this

Reply:
I think this is something that needs to be merged. The metrics.md is out of date. I've pulled upstream master and it still has this diff.